src/nginx/conf/nginx-config.template

```diff
@@ -280,6 +286,12 @@ http {
       proxy_set_header X-Forwarded-Port $forwarded_port;
       proxy_set_header X-Forwarded-Proto $scheme;
 
+        {{ if isTrue (env "ROUTER_ENABLE_UNSAFE_ANNOTATIONS") }}
```

Comment:
Could you move this block (lines 289-293) to the end of the location https://github.com/nginxinc/nginx-openshift-router/blob/c893a551cc013a4c472affb4158789d1e5bc2d74/src/nginx/conf/nginx-config.template#L316 and insert it after the last `{{ end }}` before `}` ?


 Otherwise, gRPC locations, controlled by the following if https://github.com/nginxinc/nginx-openshift-router/blob/c893a551cc013a4c472affb4158789d1e5bc2d74/src/nginx/conf/nginx-config.template#L241 will not be supported.

Reply:
Adjusted in 9cbb472